index.html

```diff
@@ -29,5 +29,122 @@ <h4>CONNECT WITH US</h4>
             <li><a href="#"><i class="fab fa-medium-m"></i></a></li>  
         </ul>
     </section>
+
+    <div class="works-grid">
+        <section class="work-1">
+            <div class="work-img"><img src="./assets/images/Snapshoot-Portfolio.png" alt="Portfolio snapshoot"></div>
+            <div class="work-desc"><h1>Tonic</h1>
+            <ul class="frame">
+                <li class="canopy">CANOPY</li>
+                <li class="b-e-d">Back End Dev</li>
+                <li class="year">2022</li>
+            </ul>
+            <p  class="work1-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="framework">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+            </div>        
+        </section>
```

Comment:
- [ ] Make sure to use the `section` tag for the container of the **works** section, which will improve your project to look more semantic you can check [this link](https://www.w3schools.com/html/html5_semantic_elements.asp#:~:text=A%20semantic%20element%20clearly%20describes,%3E%20%2D%20Clearly%20defines%20its%20content.) for more detail